src/desktop/assets/analytics.ts

```diff
@@ -42,26 +41,7 @@ require("../analytics/main_layout.ts")
 
 $(() =>
   window.analytics.ready(function () {
-    if (sd.CURRENT_USER != null ? sd.CURRENT_USER.id : undefined) {
-      const allowedlist = [
-        "collector_level",
-        "default_profile_id",
-        "email",
-        "id",
-        "name",
-        "phone",
-        "type",
-      ]
-      const traits = extend(pick(sd.CURRENT_USER, allowedlist), {
-        session_id: sd.SESSION_ID,
-      })
-      window.analytics.identify(sd.CURRENT_USER.id, traits, {
-        integrations: { Marketo: false },
-      })
-      // clear analytics cache when user logs out
-      analyticsHooks.on("auth:logged-out", () => window.analytics.reset())
-    }
-
+    onAnalyticsReady()
     setupSplitTests()
 
```

Comment:
Will investigate this in a separate PR but wanted to note that this `setupSplitTests` is not present in mobile-- it is possible that split tests have been skipping pages entirely if they use the legacy mobile templates.